packages/flutter/test/cupertino/scrollbar_test.dart

```diff
@@ -163,4 +163,233 @@ void main() {
     await tester.pump(_kScrollbarTimeToFade);
     await tester.pump(_kScrollbarFadeDuration);
   });
+
+  testWidgets('On first render with displayAlways: true, the thumb shows',
+      (WidgetTester tester) async {
+    final ScrollController controller = ScrollController();
+    Widget viewWithScroll() {
+      return Directionality(
+        textDirection: TextDirection.ltr,
+        child: MediaQuery(
+          data: const MediaQueryData(),
+          child: PrimaryScrollController(
+            controller: controller,
+            child: CupertinoScrollbar(
+              displayAlways: true,
+              controller: controller,
+              child: const SingleChildScrollView(
+                child: SizedBox(
+                  width: 4000.0,
+                  height: 4000.0,
+                ),
+              ),
+            ),
+          ),
+        ),
+      );
+    }
+
+    await tester.pumpWidget(viewWithScroll());
+    await tester.pumpAndSettle();
```

Comment:
I assume this doesn't work without the pumpAndSettle because we delay drawing the scrollbar for one frame.  Could you replace it with just a `pump`?  That way it's a little more strict and it shows that the scrollbar isn't fading in.

Reply:
Maybe add a comment here explaining the need for the extra pump/pumpAndSettle.  Here and in the other relevant tests.

Reply:
> I assume this doesn't work without the pumpAndSettle because we delay drawing the scrollbar for one frame. Could you replace it with just a `pump`? That way it's a little more strict and it shows that the scrollbar isn't fading in.

If I change with just a `pump` then the test will fail as your gues. I didn't get the point.
Should I use multiple `pump`'s?

Reply:
I mean could you replace the pumpAndSettle on line 193 with this?

```dart
await tester.pump();
```

The scrollbar isn't fading in, it should just appear on the second frame, so I think that this should work and the tests should pass.  If not though, I guess we have to use `pumpAndSettle`.

Then, add a comment to explain whey the pump is there, like this: `The scrollbar measures its size on the first frame and renders starting in the second, so pump a frame to allow it to appear.`

Reply:
with a single `pump` it is not passing, that's what I'm using `pumpAndSettle`